content/providers/provider-terraform/migrate-provider-tf.md

```diff
@@ -0,0 +1,256 @@
+---
+title: "Migrate from Terraform to Crossplane or Upbound"
+weight: 1
+description:
+---
+
+Crossplane and Terraform are powerful tools for managing and provisioning cloud
+resources. Both offer consistent infrastructure provisioning through declarative
+configurations and support multiple cloud providers for more resilient
+multi-cloud strategies. Both tools also extend beyond traditional infrastructure
+management and allow users to create and manage various resource types with
+provider plugins.
+
+<!-- vale Microsoft.HeadingPunctuation = NO -->
+
+## Why Crossplane?
+
+<!-- vale Microsoft.HeadingPunctuation = YES -->
+
+While both Crossplane and Terraform help teams manage resources, Crossplane has
+several features that may fit better with your deployment workflow.
+
+The Control Plane is the centralized management layer that orchestrates and
+coordinates the provisioning and lifecycle management of cloud resources.
+Crossplane leverages Kubernetes as its control plane, utilizing the Kubernetes
+API to manage infrastructure and resources. If your organization already uses
+Kubernetes, Crossplane allows you to simplify your workflow and minimizes tool
+sprawl.
+
+Crossplane also manages state for you as a Kubernetes Controller. If your
+external resource changes outside of the desired Crossplane configuration,
+Crossplane automatically reconciles the resource for you.
+
+## Migration prerequisites
+
+If you have resources in Terraform and want to migrate to Crossplane, make sure
+you have the following prerequisites:
+
+- A Kubernetes cluster
+- Crossplane installed
+
+- A cloud provider account
+
+This guide reviews a small Terraform configuration using AWS and use
+Crossplane's `provider-terraform` to migrate the resources to a new control
+plane.
+
+The Terraform configuration you'll work with creates a new virtual machine:
+
+```hcl
+	resource "aws_instance" "my_vm" {
+    	ami                   	= "ami-065deacbcaac64cf2"
+    	instance_type         	= "t2.micro"
+    	tags = {
+        	Name = var.vmName,
+    	}
+  	}
+	variable "vmName" {
+    	description = "VM name"
+    	type    	= string
+  	}
+```
+
+`provider-terraform` is a Crossplane provider that parses and executes your
+Terraform configurations as a Crossplane Managed Resource(MR). You don't have to
+rewrite all your Terraform configurations to begin working with Crossplane.
+
+## Create a managed resource
+
+Crossplane managed resources are declarative specifications that define your
+desired infrastructure and resources. These configurations are a Kubernetes
+manifest YAML file. Crossplane uses them to create and manage resources within
+your Kubernetes cluster.
+
+```yaml
+apiVersion: tf.upbound.io/v1beta1
+kind: Workspace
+metadata:
+  name: tf-vm
+spec:
+  forProvider:
+	source: Inline
+	module: |
+
+	###
+
+	vars:
+  	- key:
+    	value:
+```
```

Comment:
yaml is missing indentation, it's best to run yaml examples through https://github.com/adrienverge/yamllint